### DISCUSSION TITLE
What is REPL

### DISCUSSION THREAD

**Discussion by jborbik** [mfussenegger/nvim-dap#1095] [Q&A] [answered]
Could someone ELI5 what is the function of dap-repl? Can I use it to evaluate expressions?

**Comment by shmerl** [comment, marked as answer]
[REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop) is the interactive prompt of the debugger where you can execute commands like you'd do inside regular gdb. You need to prefix them with `-exec ` though.
